docs/release-notes/8.3.asciidoc

```diff
@@ -0,0 +1,82 @@
+* Adds “Vulnerable hosts by severity” and “Vulnerable users by severity” tables to the Detection & Response page ({pull}128335[#128335]).
+* Adds an “Open alerts by rule” table that shows up to four rules sorted by severity and then by alert count to the Detection & Response page ({pull}129021[#129021]).
+* Adds an “Alerts” widget to the Detection & Response page with donut charts that show alerts by status and severity ({pull}128087[#128087]).
+* Enhances Detection & Response page widgets by adding pagination and making numbers you click open directly in timeline ({pull}131828[#131828]).
+* Adds a “Recently created cases” widget to the Cases page ({pull}131029[#131029]).
+* Creates landing pages for Dashboards and Threat hunting, for now only accessible via URL ({pull}130905[#130905]).
+* Adds a “User risk” tab to the User details flyout ({pull}130256[#130256]).
+* Adds an “Authentications” tab to the User details flyout ({pull}129456[#129456]).
+* Adds the ability to investigate Osquery results in Timeline ({pull}128596[#128596]).
+* Allows multiple alerts to be added to a case ({pull}130958[#130958]).
+* Adds the option to delete case comments multiple alerts to be added to a case ({pull}130254[#130254]).
+* Provides an option to select a severity level for a case ({pull}131626[#131626]).
+* Adds the *Alerts* tab to cases ({pull}131883[#131883]).
+* Adds the *Average time to close* metric to the Cases page ({pull}131909[#131909]).
+
+[discrete]
+[[bug-fixes-8.3.0]]
+==== Bug fixes and enhancements
+* Separates array values in the Alerts table with commas ({pull}133297[#133297]).
+* Exposes the EQL search configurations `event_category_field`, `tiebreaker_field`, and `timestamp_field` through the rules API and UI for Event Correlation rules ({pull}132247[#132247]).
+* Adds the *Session ID* field to the *Highlighted fields* section on the Alert details flyout ({pull}132219[#132219]).
+* Introduces a new design for the main navigation menu ({pull}132210[#132210], {pull}131437[#131437]).
+* Adds Dashboards and Threat Hunting Landing pages ({pull}130905[#130905]).
+* Introduces the Detections & Response dashboard ({pull}130670[#130670], {pull}129021[#129021], {pull}131029[#131029], {pull}128335[#128335], {pull}128087[#128087]).
+** Update Detections Response view with pagination and opening numbers in Timeline ({pull}131828[#131828], {pull}131029[#131029]).
+* Allows highlighted fields to be investigated in Timeline ({pull}131255[#131255]).
+* Adds support to Osquery for multi-line queries ({pull}131224[#131224]).
+* Enforces the `All` global privilege setting on Osquery ({pull}130523[#130523]).
+* Adds the *Run Osquery* option to the *More actions* menu (*...*) in the Alerts table ({pull}131790[#131790]).
+* Improves the performance of these actions on the bulk rule actions endpoint ({pull}130924[#130924]).
+** `add_tags`
+** `delete_tags`
+** `set_tags`
+** `add_index`
+** `delete_index`
+** `set_index_patterns`
+** `set_timeline_template`
```

Comment:
I think these are not correct. These are the constants we have in the code:

```ts
export enum BulkActionEditType {
  'add_tags' = 'add_tags',
  'delete_tags' = 'delete_tags',
  'set_tags' = 'set_tags',
  'add_index_patterns' = 'add_index_patterns',
  'delete_index_patterns' = 'delete_index_patterns',
  'set_index_patterns' = 'set_index_patterns',
  'set_timeline' = 'set_timeline',
}
```

@nastasha-solomon so it should be:

```
** `add_index_patterns`
** `delete_index_patterns`
** `set_index_patterns`
** `set_timeline`
```

It also turns out that the docs are not correct, because they say `add_index` etc:
https://www.elastic.co/guide/en/security/current/bulk-actions-rules-api.html#_possible_bulkeditaction_object_values

@vitaliidm could you please fix the docs?

Reply:
@nastasha-solomon I ended up fixing it in https://github.com/elastic/security-docs/pull/2130